style.css

```diff
@@ -0,0 +1,201 @@
+    flex-direction: column;
+    justify-content: space-evenly;
+    right: 36.612px;
+    top: 40px;
+
+    /* border: red solid 3px;  */
+  }
+
+  .bar {
+    display: block;
+    position: relative;
+    width: hug(16px);
+    height: 2px;
+    background-color: var(--Primary-text-color);
+  }
+
+  .HamburgerX {
+    display: none;
+    position: absolute;
+    cursor: pointer;
+    width: 18px;
+    height: 1.2rem;
+    flex-direction: column;
+    justify-content: space-evenly;
+    right: 36.612px;
+    top: 20px;
+
+    /* border: red solid 3px;  */
+  }
+
+  .barX {
+    display: none;
+    position: relative;
+    width: hug(16px);
+    height: 2px;
+    background-color: var(--Primary-text-color);
+  }
+
+  .welcome {
+    width: hug(5.625rem);
+    height: hug(2rem);
+    font-family: "Inter", 'Courier New', Courier, monospace;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1.4rem;
+    line-height: 1.95rem;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #28352f;
+    margin-left: 1rem;
+  }
+
+  .Menu-items {
+    display: none;
+    position: absolute;
+    height: 100px;
+    width: 30%;
+    left: 20px;
+    top: 98px;
+  }
+
+  .nav--link {
+    /* Theme 2 / H3 */
+
+    font-family: "Inter", 'Courier New', Courier, monospace;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    padding-bottom: 4rem;
+    display: flex;
+    align-items: center;
+    color: var(--Primary-text-color2);
+
+    /* End of Navbar */
+  }
+
+  /* Header section */
+  .header {
+    width: 100%;
+    height: 100%;
+    padding-bottom: 10rem;
+    background: url("images/Mob-main.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .header1 {
+    margin-top: 4.5rem;
+    width: 20.4rem;
+    height: 12rem;
+    font-family: "Inter", 'Courier New', Courier, monospace;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.6rem;
+    line-height: 3.2rem;
+    letter-spacing: 1.5px;
+    text-align: center;
+    color: var(--Primary-text-color);
+  }
+
+  .header2 {
+    /* margin-top: -5.5rem; */
+    width: 20.4rem;
+    height: 12rem;
+    font-family: "Inter", 'Courier New', Courier, monospace;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.2rem;
+    letter-spacing: 2px;
+    text-align: center;
+    color: var(--buttons-color);
+  }
+
+  .header-detail {
+    margin-top: 4rem;
+    margin-bottom: 5rem;
+    width: 22.29rem;
+    height: 164px;
+    font-family: "Inter", 'Courier New', Courier, monospace;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.4rem;
+    line-height: 1.95rem;
+    text-align: center;
+    color: var(--Primary-text-color);
+  }
+
+  .social {
+    width: 40%;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
### Without align-items: center;
![without-align-items (copy)](https://user-images.githubusercontent.com/93039359/184948631-3e5f3a0e-1719-4fed-a050-fa6b408cc0ff.png)

### With align-items: center;
![with-align-items (copy)](https://user-images.githubusercontent.com/93039359/184948663-4eb6c894-8daa-4fc3-b22c-b99d895c314c.png)

Please add align-items: center; to make your icons align in center vertically to follow the Figma template :+1: